examples/plot_virtual_evoked.py

```diff
@@ -0,0 +1,33 @@
+"""
+=======================
+Remap MEG channel types
+=======================
+
+In this example, MEG data are remapped from one
+channel type to another. This is useful for combining statistics
+for magnetometer and gradiometers. This process can be
+computationally intensive.
+"""
+
+# Author: Mainak Jas <mainak.jas@telecom-paristech.fr>
+
+# License: BSD (3-clause)
+
+from mne.datasets import sample
+from mne import read_evokeds
+from mne.forward import compute_virtual_evoked
+
+print(__doc__)
+
+data_path = sample.data_path()
+evoked_fname = data_path + '/MEG/sample/sample_audvis-ave.fif'
+
+from_type, to_type, condition = 'grad', 'mag', 'Left Auditory'
+evoked = read_evokeds(evoked_fname, condition=condition, baseline=(-0.2, 0.0))
+
+virtual_evoked = compute_virtual_evoked(evoked, from_type=from_type,
+                                        to_type=to_type)
```

Comment:
going from grad to mags appears to work fine

![grad_to_mag](https://cloud.githubusercontent.com/assets/3817535/7103798/819791e8-e0b9-11e4-819b-89581d392a88.png)

but the other way round (mag to grad) looks a bit suspicious to me

![mag_to_grad](https://cloud.githubusercontent.com/assets/3817535/7103801/8c8d6f64-e0b9-11e4-9992-67b5191a7ca0.png)

Fig 1 is True mag/grad whereas Fig 2 is interpolated